smallcloudai/refact-chat-js
# Fix/paths with spaces

# Fix pin paths with spaces

## Description

The file path can have a space in it, this would cause file path in the pin message to be cut of early.

There's been an lsp update to fix this here https://github.com/smallcloudai/refact-lsp/pull/414

this has be written to be backward compatible with older messages.

## Type of change


## How to Test



- Open an old pin message, it should still look the same,
- Create a new pin message, it should work when applying

## Screenshots (if applicable)

No viable changes

## Checklist


## Linked Issues
relies on 
https://github.com/smallcloudai/refact-lsp/pull/414

## Additional Notes
